src/elementPlugins/shortText/src/SummaryView.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Header, Icon } from 'semantic-ui-react'
+import { SummaryViewProps } from '../../types'
+
+const SummaryView: React.FC<SummaryViewProps> = ({ parameters, value }) => {
+  return (
+    <>
```

Comment:
better to use a Container here

Reply:
Yeah, but what Semantic component would you suggest? There didn't seem to be an appropriate one. Something like `Segment.Group` would seem appropriate around the whole list, but not sure what for individual elements. If there's no distinguishing boundary between them, does it need it?

Reply:
I think the `Container` component is good. It doesn't add any spaces in the group it contains, I think that's alright for now.

Reply:
Container is for containing groups of elements, I'm not sure I understand why we'd want to wrap single elements in their own container.

I guess we should probably start making some standards/rules/style guides for this stuff cos I can see it getting very confusing and inconsistent. Things like making sure "Label" always gets a "Header" component (for example, probably not in actuality) and "Description" is always body text, etc. And what kind of Groups and Containers we wrap things in, etc. I guess we'll probably wait till the designs are close to the final versions to do that, right?

Reply:
> I'm not sure I understand why we'd want to wrap single elements in their own container.

Reply:
Trying Segment and Segment.Group, but they both style each question in their own "box", which is not what we want.

I think of Container in terms of Flexbox, and I wouldn't put Container around Flexbox Items unless they, in turn, contained things that needed formatting distinct from the parent.

I think I'll just leave as is now -- have made an issue to discuss these issues: https://github.com/orgs/openmsupply/projects/6#card-49947515